h2o-r/tests/testdir_algos/gbm/runit_GBM_bernoulli.R

```diff
@@ -44,17 +44,22 @@ test.GBM.bernoulli <- function() {
   Log.info("R Confusion Matrix:")
   print(RCM)
   Log.info("H2O Confusion Matrix:")
-  print(h2o.confusionMatrix(h2o.performance(prostate.h2o)))
+  perf <- h2o.performance(prostate.h2o)  
+  print(h2o.confusionMatrix(perf))
   
   R.auc <- gbm.roc.area(prostate.data$CAPSULE,R.preds)
   Log.info(paste("R AUC:", R.auc, "\tH2O AUC:", h2o.auc(h2o.performance(prostate.h2o))))
 
   # PUBDEV-515
```

Comment:
```suggestion
  #13556
```

Reply:
I don't think this change should be part of this PR. I added something new to this test, and this comment is completely unrelated.

Reply:
All right, I'll do it in separete PR, its missed point from conversion to GH.